### PR TITLE
(GH-309) Display Addins in alphabetical order

### PR DESCRIPTION
**Pull request by tomaszbartoszewski**

### CODE REVIEW

**Comment by tomaszbartoszewski** [comment]
Hi, this is the easiest way to get it done, but if you wanted something better, feel free to reject this PR.

**Comment by devlead** [comment]
Relates to #309

File: input/_AddinsList.cshtml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <p>This page contain third party addins for Cake. If you want to file an issue for any addin listed here, contact the author of the addin.</p>
 <p>If you're an addin author and want your addin listed here, follow the instructions <a href="https://github.com/cake-build/website/blob/develop/README.md#addins">here</a>.</p>
 
-@foreach(IDocument addin in Model)
+@foreach(IDocument addin in Model.OrderBy(d => d.String("Name")))
```
Comment (devlead) [inline input/_AddinsList.cshtml]: LGMT 👍  Might improve with `StringComparer.OrdinalIgnoreCase` in future.

**Comment by devlead** [comment]
@tomaszbartoszewski your changes have been merged, thanks for your contribution 👍

**Pull request merged**